app/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source.rb

```diff
@@ -33,4 +36,49 @@ def sync
     update!(:status => "error", :last_updated_on => Time.zone.now, :last_update_error => error)
     raise error
   end
+
+  private
+
+  # Find Terraform Templates(dir) in the git repo.
+  # Iterate throught git repo worktree, and collate all terraform template dir's (dirs with .tf or .tf.json files).
+  #
+  # Returns [hashmap] of template dirs and files within it.
+  def find_templates_in_git_repo()
+    template_dirs = {}
+
+    # checkout files to temp dir, we need for prasing input/output vars
+    git_checkout_tempdir = Dir.mktmpdir("terraform-git")
+    checkout_git_repository(git_checkout_tempdir)
+
+    # traverse through files in git-worktree
+    git_repository.with_worktree do |worktree|
+      worktree.ref = scm_branch
+
+      # Find all dir's with .tf/.tf.json files
+      worktree.blob_list.each do |filepath|
+        next unless filepath.end_with?(".tf", ".tf.json")
+
+        parent_dir = File.dirname(filepath)
+        next if template_dirs.has_key?(parent_dir)
+
+        full_path = File.join(git_checkout_tempdir, parent_dir)
+        _log.info("Local full path : #{full_path}")
+        files = Dir.children(full_path)
+
+        # :TODO add parsing for input/output vars
+        input_vars = nil
+        output_vars = nil
+
+        template_dirs[parent_dir] = {
+          :relative_path => parent_dir,
+          :files         => files,
+          :input_vars    => input_vars,
+          :output_vars   => output_vars
+        }
+        _log.debug("=== Add Template:#{parent_dir} for #{filepath}")
+      end
+    end
+    template_dirs
+  end
+
```

Comment:
@putmanoj `app/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source.rb:83:1: C: [Correctable] Layout/EmptyLinesAroundClassBody: Extra empty line detected at class body end.`
```suggestion
```

Reply:
I saw this on with @sudhir-kelkar 's PR as well, is this an editor setting that you guys have?

Reply:
@agrare 

How do we setup the editor ? don't remember seeing doc ...,  do you have doc link you can share

Reply:
@putmanoj there are not docs on setting up your editor since your editor is your choice.  If your editor is automatically changing code that isn't in line with the org's rubocop rules that is up to you to fix.